client/my-sites/stats/post-trends/style.scss

```diff
@@ -163,55 +197,117 @@
 	}
 
 	&.is-hovered {
-		border-color: var(--color-primary);
+		transform: scale(1.2);
 	}
 
 	.is-loading & {
 		animation: loading-fade 1.6s ease-in-out infinite;
 	}
 }
 
-@include breakpoint-deprecated( ">1040px" ) {
+@include breakpoint-deprecated(">1040px") {
+
 	.post-trends__scroll-left,
 	.post-trends__scroll-right {
 		display: none;
 	}
 }
 
 .post-trends__key-container {
-	@include clear-fix;
-	padding-top: 10px;
-	float: right;
-
-	@include breakpoint-deprecated( "<960px" ) {
-		float: none;
-		margin: 0 auto;
-		margin-bottom: 25px;
-		text-align: center;
+	margin-top: 16px;
+	float: none;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+
+	@media (max-width: $break-large ) {
+		margin-bottom: 0;
 	}
 }
 
-.post-trends__key-label,
-.post-trends__key {
-	display: inline-block;
-}
-
 .post-trends__key-label {
-	font-size: $font-body-extra-small;
-	color: var(--color-neutral);
-	letter-spacing: 0.1em;
-	text-transform: uppercase;
+	font-weight: 400;
+	font-size: $font-body-small;
+	line-height: 18px;
+	color: #000;
```

Comment:
Let's use the corresponding CSS variable for this color (`--color-text`, perhaps?)

Reply:
@wilmersondasilva, this comment needs a follow-up.